docs/resources/compensating-research-participants.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Compensating research participants
+parent: Resources
+nav_order: 6
+---
+
+# Compensating research participants
+{: .no_toc .fw-700 }
+
+How do we compensate folks for their time in our design research activities? There are plenty of reasons to do so…
+- [Compensating research participants @ 18F](https://ux-guide.18f.gov/research/plan/#compensating-research-participants)
+- [Equity centered design @ Digital.gov](https://digital.gov/files/equity-centered-design-julieben.pptx)
+- [Ethical and practical considerations of paying research participants](https://www.niehs.nih.gov/research/resources/assets/docs/ethical_and_practical_considerations_of_paying_research_participants_508.pdf)
+- [Why Design Researchers Should Compensate Participants](https://medium.com/notes-off-the-grid/why-design-researchers-should-compensate-participants-a65252352f67)
+- [The influence of risk and monetary payment](https://jme.bmj.com/content/30/3/293)
+- [Go beyond extractive "do no harm" to produce research that "heals"](https://twitter.com/albanvillamil/status/1311687313802645504)
+- [Qualitative Research at Code for America](https://info.codeforamerica.org/qualitative-research)
+
+...but it can be challenging making this work on our projects.
+
+## Government employees
+We generally [**cannot** compensate government employees](https://www.doi.gov/ethics/gifts) due to anti-corruption measures. 
+
+We [**can** offer them](https://ogc.commerce.gov/sites/default/files/gifts-awae-2020.pdf) a modest non-cash gift valued less than $20 like a snack, trophy/challenge coin or certificate, [drawings of agency leadership in costumes](https://cmsgov.slack.com/archives/CHPGY8H9V/p1625008393044400), t-shirt, or book related to their agency or profession. The employee may still decline, just in case, so it’s worth offering up front instead of surprising them, and they are limited to $50 per year from the same donor.
```

Comment:
The link to costumes (which I desperately wanted to click on) led me to CMS Slack login, which I do not have. Might not be worth including the link.

Reply:
oh yeah i got that link back when i was on a CMS project, so I'll just remove it. But it was a nice story that this is a fair paraphrasing of for future reference :)